openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -202,6 +202,22 @@ def get_all_orgs():
     return site_configuration_orgs.union(microsite_orgs)
 
 
+def get_site_domain_for_org(org, default=None):
+    """
+    This returns site domain value which has an org_filter that matches
+    what is passed in,
+
+    Args:
+        org (str): Course org filter, this value will be used to filter out the correct site configuration.
+        default: default value to return if key is not present in the configuration
+
+    Returns:
+        Site domain for the given org.
+
+    """
+    return SiteConfiguration.get_site_domain_for_org(org, default)
```

Comment:
I don't think we need this function. We can just call the class method on the SiteConfiguration model.